core/modules/qdisp/QueryRequest.cc

```diff
@@ -153,6 +154,13 @@ bool QueryRequest::_importStream() {
     bool retrieveInitiated = false;
     // Pass ResponseRequester's buffer directly.
     std::vector<char>& buffer = _requester->nextBuffer();
+    LOGF_DEBUG("QueryRequest::_importStream buffer.size=%1%" % buffer.size());
+    for(auto iter=buffer.begin(); iter != buffer.end(); iter++) *iter=8;
```

Comment:
Our guidelines recommend multi-line for:

    for(auto iter=buffer.begin(); iter != buffer.end(); iter++) {
        *iter=8;
    }

Also prefer `++ iter` over `iter ++`.

Is 8 some magical number everyone is supposed to know about?